### PR TITLE
Move to using Rems for breakpoints

### LINKED ISSUE #361: SitewideLinks Component doesnt expand to full group 5 width
**Describe the bug**
SitewideLinks Component doesn't expand to full group 5 width

**To Reproduce**

1. Go to any news article
2. See error

**Expected behavior**
It expands to the full 1280px

**Screenshots**
image

**Additional context**

- [x] Initially labelled with ["bug"](https://github.com/bbc/psammead/labels/bug)

### PR DESCRIPTION
**Pull request by dr3**
Part-resolves #361

**Overall change:** Move to using Rems for breakpoints so that the width of a component isnt influenced by the font size of its container 

**Code changes:**

- Move to using Rems for breakpoints
- Updates Readme

**Testing:**
This can be tested by linking the gel foundations component to simorgh and then using ngrok and browserstack to do a visual regression on different browsers, taking support https://caniuse.com/#feat=rem into consideration.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

File: packages/utilities/gel-foundations/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ const SomeStyledComponent = css`
 
 To allow the typography to be fully accessible and responsive, please note that you should apply a default font-size to the document root (e.g. `html { font-size: 100% }`).
 
-Our typography uses `em` for font-size and `rem` for line-height. `em` allows modularity of components: you can change the component font-size by changing the font-size of its container. `rem` is relative to the document root, so we use that for line-height and spacing for a consistent look-and-feel across the document. You can read our [detailed analysis of "REMs vs EMs for spacing"](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md) for more information.
+Our breakpoints and typography font-size and line-heights use `rem` for sizing due to the charity and reduced side effects in its usage. `rem` is relative to the document root, so we use that for line-height and spacing for a consistent look-and-feel across the document. You can read our [detailed analysis of "REMs vs EMs for spacing"](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md) for more information.
```
Comment (sareh) [inline packages/utilities/gel-foundations/README.md]:
```suggestion
Our breakpoints and typography font-size and line-heights use `rem` for sizing due to the reduced side effects in its usage. `rem` is relative to the document root, so we use that for line-height and spacing for a consistent look-and-feel across the document. You can read our [detailed analysis of "REMs vs EMs for spacing"](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md) for more information.
```

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.1.0   | [PR#360](https://github.com/BBC/psammead/pull/360) Move to using rems for breakpoints |
```
Comment (sareh) [inline packages/utilities/gel-foundations/CHANGELOG.md]:
```suggestion
| 0.3.0   | [PR#360](https://github.com/BBC/psammead/pull/360) Move to using rems for breakpoints |
```

Comment (dr3) [reply packages/utilities/gel-foundations/CHANGELOG.md]: Whoops 🤦‍♂️

**Comment by sareh** [comment]
We should ensure we add notes for testers to link this work in with Simorgh so they can do browser regression. Here is the browser support for rems https://caniuse.com/#feat=rem

**Comment by dr3** [comment]
Done @sareh :)

**Review by sareh — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Comment by david-boydell** [comment]
LGTM

**Pull request merged**